tests/integration_rust/snapshots/integration_rust__upgrade_tests__pypi_dependency_index_preserved_on_upgrade.snap

```diff
@@ -1,14 +1,14 @@
 ---
 source: tests/integration_rust/upgrade_tests.rs
-expression: "fs_err::read_to_string(file_path).unwrap_or_default()"
+expression: pixi.manifest_contents().unwrap_or_default()
 ---
-[workspace]
-channels = ["conda-forge"]
-platforms = ["linux-64", "osx-arm64", "win-64"]
-exclude-newer = "2025-05-19"
+        [workspace]
+        channels = ["conda-forge"]
```

Comment:
This changed because we are using different mechanisms for reading back the toml.